network/src/web.ts

```diff
@@ -0,0 +1,77 @@
+import { WebPlugin, PluginListenerHandle } from '@capacitor/core';
+import { NetworkPlugin, NetworkStatus } from './definitions';
+
+declare var window: any;
+
+export class NetworkWeb extends WebPlugin implements NetworkPlugin {
+  listenerFunction: any = null;
+
+  constructor() {
+    super({ name: 'Network' });
+  }
+
+  getStatus(): Promise<NetworkStatus> {
+    return new Promise((resolve, reject) => {
+      if (!window.navigator) {
+        reject('Network info not available');
```

Comment:
This should use `UnsupportedBrowserException` with a specific message about what browser API is missing, see https://github.com/ionic-team/capacitor-plugins/blob/main/screen-reader/src/web.ts#L17-L19 for an example.